lib/menu.rb

```diff
@@ -0,0 +1,22 @@
+class Menu
+    attr_reader :list, :choices
+    def initialize 
+        @list = [
+            {:dish => 'green curry', :price => 5.99},
+            {:dish => 'panang', :price => 7.5},
+            {:dish => 'veg tempura', :price => 4.5},
+            {:dish => 'rice', :price => 2.5}
+            ]
+        @choices = []
+    end
+
+    def select(dish_string, quantity)
```

Comment:
Not sure that the class menu has the responsibility of select the dishes. 
Maybe an other class like order or basket would have been a way to approach the challenge